charts/outdated/values.yaml

```diff
@@ -116,19 +122,6 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
-# Additional volumes on the output Deployment definition.
```

Comment:
We often keep these in the Helm charts because they might be useful for future operators that have unique requirements depending on their runtime env.

One such case would be being able to inject an additional CA file into the Pod. There are also some Kubernetes Secrets integrations like CSI Secrets Provider that rely on being able to add volumes and volumeMounts to a container.

In the volume this would look something like this:
```yaml
      volumes:
        - name: repos
          persistentVolumeClaim:
            claimName: {{ include "outdated.fullname" . }}
      {{- with .Values.volumes }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
```
Given the helm create defaults are rather nice in the regards, we usually keep it, otherwise this is for sure something that can be implemented when it's needed.